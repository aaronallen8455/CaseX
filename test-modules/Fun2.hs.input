module LambdaCase1 where

import qualified Reexport as R

main :: IO ()
main = pure ()

test :: R.Foo -> Bool
test SPLIT = True
