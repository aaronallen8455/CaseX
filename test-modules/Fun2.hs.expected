module LambdaCase1 where

import qualified Reexport as R

main :: IO ()
main = pure ()

test :: R.Foo -> Bool
test R.Foo = True
test R.Bar = True
test R.Baz = True
